### ISSUE TITLE
Which expressions are available?

### ISSUE THREAD

**Issue by jpiabrantes** [albertodev01/equations#39]
I am trying to solve a non-linear equation and getting the error:
`Error: ExpressionParserException: The given expression cannot be parsed! Make sure that all operators are supported. Make also sure that the product of two values explicitly has the '*' symbol.`

How do I find which expressions are available?

Here's my code:
```dart
final newton = Newton(function: "0.03606737602222409*x*log(exp((3537.428612970299+x)/287.171070460561)+4914469.74557367)-15.455695470480395", x0: 5);
newton.solve();
```

**Comment by jpiabrantes** [comment]
this might be a bug, if I replace exp with cos everything works. Why isn't exp working?

**Comment by albertodev01** [comment]
I will investigate this by this weekend, thank you for reporting the issue 🙂 

I also should include in the docstring the full operators list

**Comment by jpiabrantes** [comment]
I was able to fix it by adding to the ExpressionParser
```dart
..wrapper(
      string('exp(').trim(),
      char(')').trim(),
          (_, a, __) => (value) => math.exp(a(value)),
      )
```

happy to send a PR if the repo is open to that.

**Comment by albertodev01** [comment]
Yeah it looks like I forgot to add the `exp` function to the grammar! I have to add other features to the package so I will include this fix as well 😄 

For now, you can workaround this using `e^(...)` instead of `exp(...)`

**Comment by albertodev01** [comment]
I am closing this since this bug is gone and the fix will be part of the next `4.1.0` 😄
